packages/components/src/Text/Text.tsx

```diff
@@ -50,7 +52,14 @@ export const Text: PolymorphicComponentWithRef<TextOwnProps, 'span'> =
         {...props}
         as={as}
         variant={`text.${variant}`}
-        css={{ textAlign: align, color, cursor, outline, userSelect }}
+        css={{
```

Comment:
@sebald should these props be able to be overwritten by the variant or not? 🤔

Reply:
I think not ... I hope not :D (since its `undefined`)